04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -314,7 +320,7 @@ george.destroy
 ## Associations
 
 ### Reframing (10 / 165)
-**Question:** We have a lot of choice when it comes to databases, why did we choose to use SQL?
+**Question:** We have a lot of choice when it comes to databases, are we using a SQL db in PostgreSQL?
```

Comment:
I'm not that into, in that I'm not exactly sure what it's asking (SQL vs NoSQL?) We talked a lot about that yesterday...